resources/assets/uad_lists.json

```
@@ -37664,9 +37724,219 @@
     "id": "com.google.android.carrier",
     "list": "Oem",
     "description": "Carrier Settings\nGoogle Api Activity, requires GMS, totally random api's.\nNot needed app, a lot metrics or telemetry frameworks.",
+    "id": "com.oplus.notificationmanager",
+    "list": "Oem",
+    "description": "Notification managment will not work.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Unsafe"
+  },
+  {
+    "id": "com.oplus.ocloud",
+    "list": "Oem",
+    "description": "It's not about cloud but a lot of logs.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.oplus.omoji",
+    "list": "Oem",
+    "description": "Omoji\nAvatar editing app.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.android.bluetooth.oplus.overlay",
+    "list": "Oem",
+    "description": "Overlay to 'com.android.bluetooth'.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Unsafe"
+  },
+  {
+    "id": "com.android.camerabigballconfig.overlay",
+    "list": "Oem",
+    "description": "Configures default Qr Code component\ncom.google.android.gms .mlkit.barcode.ui.PlatformBarcodeScanningActivityProxy. Not needed to camera or scan QR Code.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.oplus.engineercamera",
+    "list": "Oem",
+    "description": "EngineerCamera\nTesting camera things.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.oplus.engineermode",
+    "list": "Oem",
+    "description": "EngineerMode\nTesting phone things(like audio, bluetooth, etc.).",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.oplus.engineernetwork",
+    "list": "Oem",
+    "description": "EngineerNetwork\nTesting network things.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.oplus.exserviceui",
+    "list": "Oem",
+    "description": "Gesture_Motion Services\nIs related to gesture navigation 'com.oplus.gesture'.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
```

```suggestion
    "removal": "Expert"
```
If it's directly related to that gesture package, then it should break gestures entirely as well I think. Therefore I think `Expert` makes more sense.

---

advanced?

---

No, why? Gestures are a vital and widespread part of the system.